lib/faraday/utils.rb

```diff
@@ -95,7 +95,14 @@ def to_hash() ::Hash.new.update(self) end
 
       def parse(header_string)
         return unless header_string && !header_string.empty?
-        header_string.split(/\r\n/).
+
+        headers = header_string.split(/\r\n/)
+
+        # Find the last set of response headers.
+        start_index = headers.rindex { |x| x.match(/^HTTP\//) }
```

Comment:
That's really strange.
I haven't used Webmock cassettes but server responses should always start with the HTTP version. I suppose Webmock authors didn't think that was important and removed it from the cassette. I would suggest to open a ticket with them and double-check